cssFiles/cardfiles/cardwithdismiss.css

```diff
@@ -0,0 +1,82 @@
+.cardcontainer2{
+    position: relative;
+    border: 1rem;
+    margin: 1rem;
+    width: 250px;
+    border-radius: 10px;
+    
+  }
+  
+  .ecom-img2{
+    position: relative;
+    height: 15rem;
+    width: 100%;
+    border-radius: 5px 5px 5px 5px;
+    
+  }
+  
+  .secondary-container2{
+    display: flex;
+    gap: 1px;
+    
+  }
+  
+  .actual-price2 {
+      display: inline;
+      margin-right: 4px;
+      color: grey;
+      text-decoration: line-through;
+    }
+    .disc-price2 {
+      margin-right: 4px;
+      display: inline;
+    }
+    .offer-info2 {
+      margin-right: 4px;
+      display: inline;
```

Comment:
I have observed that you have repeated `display: inline` quite a few times in your CSS files. So maybe you can try making it a utility class and reuse it whenever needed